<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyProfile</class>
 <widget class="QWidget" name="ModifyProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>修改用户 Modify Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color: rgb(254,255,250);}

QPushButton {
	font:10pt &quot;Consolas&quot;;
	border:2px groove white;border-radius:10px;padding:2px 4px;
	background-color: rgb(255,108,0);color: rgb(255,255,255);
}

QLabel {
	font:10pt &quot;Consolas&quot;;
}

QLineEdit {
	font:10pt &quot;Consolas&quot;;
	background-color: rgb(223,227,239);
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>281</width>
     <height>152</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2_p">
      <property name="text">
       <string>密码/Password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_1_u"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2_p">
      <property name="text">
       <string>（选填）</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3_n">
      <property name="text">
       <string>（选填）</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_4_m">
      <property name="text">
       <string>（选填）</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_5_g">
      <property name="text">
       <string>（选填）</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5_g">
      <property name="text">
       <string>权限/Privilege</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4_m">
      <property name="text">
       <string>邮箱/Mail</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1_u">
      <property name="text">
       <string>用户名/Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3_n">
      <property name="text">
       <string>姓名/Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="do_modify_profile">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>87</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交/Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
